cmd/drone-ecr/main.go

```diff
@@ -114,6 +119,34 @@ func main() {
 	os.Setenv("DOCKER_PASSWORD", password)
 	os.Setenv("PLUGIN_REGISTRY_TYPE", "ECR")
 
+	// Skip if tag already exits for both mutable and immutable repos
+	if skipPushIfTagExists {
+		tagInput := getenv("PLUGIN_TAG", "PLUGIN_TAGS")
```

Comment:
Can you use the variable parsed by golang tool instead of reading directly from env?

Reply:
nvm ignore my comment